.ci/special-jobs/single-domain.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -xe
+# Make SSFW single domain
+
+# remove all domain URLs except URLs for domain with ID more than 1 in app/config/domains.yml
+sed '/id: 2/{:a;Q}' $WORKSPACE/project-base/app/config/domains_urls.yml.dist > $WORKSPACE/project-base/app/config/domains_urls.yml
+sed -i '/id: 2/{:a;Q}' $WORKSPACE/project-base/app/config/domains.yml
+
+# set "is-multidomain" property to "false" in build.xml
+sed -i 's/<property name="is-multidomain".*/<property name="is-multidomain" value="false" \/>/' $WORKSPACE/project-base/build.xml
+
+# remove all domains with id more than 1 from build_kubernetes.sh
+sed -i "/SECOND_DOMAIN_HOSTNAME/d" $WORKSPACE/.ci/build_kubernetes.sh
+sed -i "/SECOND_DOMAIN_HOSTNAME/{:a;Q}" $WORKSPACE/project-base/kubernetes/ingress.yml
```

Comment:
Same here, I would extract lines 1-13 to some sh file and then execute build-kubernetes.sh in jenkins.